src/sentry/static/sentry/app/components/panels/panelBody.tsx

```diff
@@ -9,33 +9,34 @@ import textStyles from 'app/styles/text';
 type FlexComponentProps = Omit<React.ComponentPropsWithoutRef<typeof Flex>, 'theme'>;
 
 type Props = FlexComponentProps & {
-  disablePadding?: boolean;
-  direction?: FlexComponentProps['flexDirection'];
+  flexible?: boolean;
+  withPadding?: boolean;
+  flexDir?: FlexComponentProps['flexDirection'];
```

Comment:
What's wrong with `direction`? Do we even use it?

Reply:
For some reason TypeScript was also complaining that it colliding with something on `FlexComponentProps` or props in `FlexBox`. It doesn't anymore so maybe that was a false alarm? It doesn't seem to be used anywhere though so maybe I should just remove it?

Reply:
Yes to removing it